cmd/things/main.go

```diff
@@ -1,5 +1,5 @@
 //
```

Comment:
I would suggest dropping a year from headers and contracting them to something like this:
```
// Copyright (c) Mainflux
// SPDX-License-Identifier: Apache-2.0
```

Reply:
@drasko can we do this in another PR, this one is already huge?

Reply:
Yes, sure.